sys/include/net/gnrc/tcp.h

```diff
@@ -34,13 +34,62 @@
 extern "C" {
 #endif
 
+
+/**
+ * @brief Address information for a single TCP connection endpoint.
+ * @extends sock_tcp_ep_t
+ */
+typedef struct {
+    int family;                            /**< IP Address family. */
+#ifdef MODULE_GNRC_IPV6
+    union {
+        uint8_t ipv6[sizeof(ipv6_addr_t)]; /**< IPv6 Address storage */
+    } addr;                                /**< IP Address storage */
+#endif
+    uint16_t netif;                        /**< Network interface ID */
+    uint16_t port;                         /**< Port number (in host byte order) */
+} gnrc_tcp_ep_t;
+
+/**
+ * @brief Initialize TCP connection endpoint.
+ *
+ * @param[in,out] ep          Endpoint to initialize.
+ * @param[in]     family      Address family of @p addr.
+ * @param[in]     addr        Address for endpoint.
+ * @param[in]     addr_size   Size of @p addr.
+ * @param[in]     port        Port number for endpoint.
+ * @param[in]     netif       Network inferface to use.
+ *
+ * @returns   0 on success.
+ * @returns   -EAFNOSUPPORT if @p address_family is not supported.
+ * @returns   -EINVAL if @p addr_size does not match @p family.
```

Comment:
Here and below: I think we use `@return` everywhere else.

Reply:
Done